parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -257,6 +257,32 @@ retagContainerImage() {
     fi
 }
 
+retagMCRImagesForChina() {
+    # retag all the mcr for mooncake
+    if [[ "${CONTAINER_RUNTIME}" == "containerd" ]]; then
+        # shellcheck disable=SC2016
+        allMCRImages=($(ctr --namespace k8s.io images list | grep '^mcr.microsoft.com/' | awk '{print $1}'))
+    else
+        # shellcheck disable=SC2016
```

Comment:
Does this disable the shellcheck? I remember it failing for a format like ($(...))

Reply:
I just copied this from install-deps.sh, but I think it's the $vars in single quotes https://github-wiki-see.page/m/koalaman/shellcheck/wiki/SC2016

I think it's fine as is